src/components/loader/loader.stories.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { select, boolean } from '@storybook/addon-knobs';
+import OptionsHelper from '../../utils/helpers/options-helper';
+import Loader from './loader.component';
+import Spinner from '../spinner/spinner.component';
+import { OriginalButton } from '../button/button.component';
+import { notes, info } from './documentation';
+import { notesSpinner, infoSpinner } from '../spinner/documentation';
+
+const styles = {
+  textAlign: 'left'
```

Comment:
Why is this styling being done in the story, shouldn't any required styling be in the styled component?

Reply:
it's the styling necessary for the story to display nicely in the storybook